api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/KerasConstans.kt

```diff
@@ -43,15 +43,16 @@ internal const val INITIALIZER_GLOROT_UNIFORM: String = "GlorotUniform"
 internal const val INITIALIZER_GLOROT_NORMAL: String = "GlorotNormal"
 internal const val INITIALIZER_HE_UNIFORM: String = "HeUniform"
 internal const val INITIALIZER_HE_NORMAL: String = "HeNormal"
-internal const val INITIALIZER_LECUN_UNIFORM: String = "LeCunUniform"
-internal const val INITIALIZER_LECUN_NORMAL: String = "LeCunNormal"
+internal const val INITIALIZER_LECUN_UNIFORM: String = "LecunUniform"
+internal const val INITIALIZER_LECUN_NORMAL: String = "LecunNormal"
```

Comment:
Changed names to match Keras json export of `class_name`.

Reply:
Thanks for the changes, could you please share where did you find these constants?

Reply:
Very good question. I just noticed it when I tried to import the JSON in `ModelLoaderTest.kt`.

As far as I know the Keras JSON export just uses the class names of the initializer classes.
So the reference would be in [`keras.initializers.initializers_v2.py`](https://github.com/keras-team/keras/blob/master/keras/initializers/initializers_v2.py#L749)